### PR TITLE
Updated patch for search api drush cmd

### PR DESCRIPTION
**Pull request by sharmasahil**
### Jira
https://digital-vic.atlassian.net/browse/SD-532

### Problem/Motivation
Patch breaking on previous version of module.

### Fix
Reroll patch.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -406,7 +406,7 @@
                 "[UX] Add the ability to hide or disable the import CSV option - https://www.drupal.org/project/tablefield/issues/2337743": "https://www.drupal.org/files/issues/2020-11-11/tablefield-allowed_data_sources-2337743-16.patch"
             },
             "drupal/search_api": {
-                "Provide drush support to index a specific entity - https://www.drupal.org/project/search_api/issues/3016809": "https://www.drupal.org/files/issues/2020-02-08/search_api-index-specific-items-3016809-09.patch"
+                "Provide drush support to index a specific entity - https://www.drupal.org/project/search_api/issues/3016810": "https://www.drupal.org/files/issues/2020-02-08/search_api-index-specific-items-3016809-09.patch"
```
Comment (anthony-malkoun) [inline composer.json:409]: I imagine, it's still the same issue on drupal.org?

File: composer.json [hunk 2]
```diff
@@ -37,7 +37,7 @@
         "drupal/maxlength": "^2.1",
         "drupal/entity_hierarchy": "4.x",
         "drupal/elasticsearch_connector": "^7.0",
-        "drupal/search_api": "^1.11",
+        "drupal/search_api": "^1.36",
```
Comment (anthony-malkoun) [inline composer.json:40]: ✅

File: composer.json (hunk 1 above)
Comment (anthony-malkoun) [reply composer.json:409]: This actually looks like it's using the same patch?

**Review by anthony-malkoun — CHANGES_REQUESTED** [review]
@sharmasahil I'm a bit confused about what's going on here? This worked ok in your local build?

**Comment by sharmasahil** [comment]
> @sharmasahil I'm a bit confused about what's going on here? This worked ok in your local build?

Its good now, updated both paths after a test in test branch.

File: composer.json [hunk 3]
```diff
@@ -406,7 +406,7 @@
                 "[UX] Add the ability to hide or disable the import CSV option - https://www.drupal.org/project/tablefield/issues/2337743": "https://www.drupal.org/files/issues/2020-11-11/tablefield-allowed_data_sources-2337743-16.patch"
             },
             "drupal/search_api": {
-                "Provide drush support to index a specific entity - https://www.drupal.org/project/search_api/issues/3016809": "https://www.drupal.org/files/issues/2020-02-08/search_api-index-specific-items-3016809-09.patch"
+                "Provide drush support to index a specific entity - https://www.drupal.org/project/search_api/issues/3016809": "https://www.drupal.org/files/issues/2024-11-25/3016810.patch"
```
Comment (anthony-malkoun) [inline composer.json:409]: Please follow naming pattern as per previous patch... e.g. `https://www.drupal.org/files/issues/2024-11-25/search_api-index-specific-items-3016809-10.patch`

**Review by vincent-gao — APPROVED** [review]
Thanks for the patch.

**Comment by vincent-gao** [comment]
hey @sharmasahil 
could you please fix the linting issues?

**Pull request merged**